tests/admin_filters/tests.py

```diff
@@ -263,7 +263,8 @@ def setUp(self):
         self.request_factory = RequestFactory()
 
         # Users
-        self.alfred = User.objects.create_user('alfred', 'alfred@example.com')
+        # We make requests as alfred, so we set him as superuser (so that he has change permissions on all models)
+        self.alfred = User.objects.create_superuser('alfred', 'alfred@example.com', 'xxx')
```

Comment:
`alfred` is part of the data set. I think a separate `self.superuser` would be more consistent with the other tests here. 

(Otherwise we see `request.user = self.alfred` in each method and wonder "Who's `alfred`?")

Reply:
Ok I'll do that, but this induce some changes in the expected results from the queries in the tests (the commit may be a bit confusing because of that). Let me know if you think it's worth the change.